Helper_Scripts/TTS_Installers/install_tts_omnivoice_sidecar.py

```
@@ -0,0 +1,378 @@
+    layout.runtime_dir.mkdir(parents=True, exist_ok=True)
+    layout.logs_dir.mkdir(parents=True, exist_ok=True)
+    return layout
+
+
+def validate_runtime_layout(layout: OmniVoiceRuntimeLayout) -> list[str]:
+    """Return a list of missing required runtime artifacts."""
+
+    missing: list[str] = []
+    for path in (layout.runtime_base, layout.venv_dir, layout.runtime_dir, layout.logs_dir, layout.interpreter_path):
+        if not path.exists():
+            missing.append(str(path))
```

Ruff (0.15.10)

[warning] Use a list comprehension to create a transformed list

(PERF401)